Cue/app/tabs/library/DeckView.ios.js

```diff
@@ -54,17 +54,26 @@ class DeckView extends React.Component {
       onPress: () => { this.props.navigator.pop() }
     }
 
-    const rightItems = [
-      {
-        title: 'Add Card',
-        icon: CueIcons.plus,
-        display: 'icon'
-      },
-      {
-        title: 'Edit',
-        display: 'text',
-      }
-    ]
+    let rightItems = []
```

Comment:
You can just say `let rightItems` here and then assign it inside the `if`/`else` blocks without needing to re-assign or mutate.